api/list.go

```diff
@@ -53,7 +53,12 @@ var listHandler handler = func(w http.ResponseWriter, r *http.Request, p params)
 
 	result := make([]apitypes.Entry, 0, len(entries))
 	for _, entry := range entries {
-		result = append(result, toAPIEntry(entry))
+		apiEntry := toAPIEntry(entry)
+		// Use += for efficiency
```

Comment:
What makes this more efficient? I'd have just gone `apiEntry.Path = filepath.Join(path, apiEntry.CName)`.

Reply:
But I guess that'd be wrong, because on Windows that would add a backslash.

Reply:
Thanks, that is much better. Updated.